product_docs/docs/pgd/5/cli/configuring_cli.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: "Configuring PGD CLI"
+navTitle: "Configuring PGD CLI"
+---
+
+PGD CLI can be installed on any system which is able to connect to the PGD cluster. You will require a user with PGD superuser privileges - the [bdr_superuser role](../security) - or equivalent (for example, edb_admin on BigAnimal distributed high-availability) to use PGD CLI.
+
+## PGD CLI and database connection strings
+
+You may not need a database connection string. For example, when Trusted Postgres Architect installs the PGD CLI on a system, it also configures the connection to the PGD cluster. This means that PGD CLI will automatically connect when run. 
+
+If you are installing PGD CLI manually, you must give PGD CLI a database connection string so it knows which PGD cluster to connect to. 
+
+!!! Important Setting passwords
+PGD CLI does not interactively prompt for your user's password. You must pass your password using one of the following methods:
+
+ 1. Adding an entry to your [`.pgpass` password file](https://www.postgresql.org/docs/current/libpq-pgpass.html) which includes the host, port, database name, user name, and password.
+ 1. Setting the password in the `PGPASSWORD` environment variable.
+ 1. Including the password in the connection string.
+
+We recommend the first option, as the other options don't scale well with multiple databases or compromise password confidentiality.
+!!!
+
+If you don't know the database connection strings for your PGD-powered deployment, see [discovering connection strings](discover_connections). It is a guide to finding the right connection strings for your cluster.
+
+Once you have that information, you can continue.
+
+## Configuring the database to connect to
+
+PGD CLI takes its database connection information from either the PGD CLI configuration file or the command line.
+
+
+### Using database connection strings in the command line
+
+You can pass the connection string directly to `pgd` using the `--dsn` option. For details, see the [sample use case](/pgd/latest/cli/#passing-a-database-connection-string). For example:
```

Comment:
This x-ref resolves to the top-level **EDB Postgres Distributed Command Line Interface** page.